src/lib/webpack/webpack-base-config.js

```diff
@@ -97,6 +100,37 @@ export default (env) => {
 			}
 		}),
 
+		// TypeScript
+		customConfig({
+			module: {
+				loaders: [
+					{
+						enforce: 'pre',
+						test: /\.tsx?$/,
+						use: [
+							{
+								loader: resolve(__dirname, './npm-install-loader'),
+								options: {
+									modules: ['typescript', 'awesome-typescript-loader'],
+									save: true
+								}
+							},
+							{
+								loader: 'awesome-typescript-loader',
+								options: {
+									sourceMap: true,
+									useBabel: true,
+									babelOptions: createBabelConfig(env),
+									useCache: true,
+									configFileName: tsconfig,
```

Comment:
@wub How is your experience so far with `ts-loader`. I've only used them a few months back and `awesome-typescript-loader` was a lot faster back then.

@developit I'm honestly thinking of closing this PR as now the ability to extend the webpack config has been added. That way the core cli remains lean and a config like @wub posted above is as easy as it gets.